.gitignore

```diff
@@ -21,3 +21,4 @@ lib
 !.yarn/versions
 
 github-orgs.local.yml
+product-owners.local.yml
```

Comment:
Is this a file you use for local testing? Is there a script to generate it or something?

Reply:
yep, just a local file and no there is no script to generate it. It's just a copy of the product-owners.yml file that is shipped with this repo